applications/RomApplication/tests/fluid_dynamics_test_files/test_ROM.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+import numpy as np
+
+from .MainKratosROM import TestFluidDynamicsROM
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+
+
+
+class ROMFluidDynamics(KratosUnittest.TestCase):
+#########################################################################################
+
+    def test_Fluid_Dynamic_ROM_2D(self):
+
+        with KratosUnittest.WorkFolderScope(".", __file__):
+            with open("ProjectParameters.json",'r') as parameter_file:
+                parameters = KratosMultiphysics.Parameters(parameter_file.read())
+            model = KratosMultiphysics.Model()
+            Simulation = TestFluidDynamicsROM(model,parameters)
+            Simulation.Run()
+            ObtainedOutput = Simulation.EvaluateQuantityOfInterest()
+            ExpectedOutput = np.load('ExpectedOutput.npy')
+
+            for i in range (np.shape(ObtainedOutput)[1]):
+                UP=0
+                DOWN=0
+                UP = sum((ExpectedOutput[:,i] - ObtainedOutput[:,i])**2)
+                DOWN = sum((ExpectedOutput[:,i])**2)
+                L2 = np.sqrt(UP/DOWN)
+                self.assertLess(L2, 1e-12)
```

Comment:
```suggestion
                UP=0
                DOWN=0
                up = sum((ExpectedOutput[:,i] - ObtainedOutput[:,i])**2)
                down = sum((ExpectedOutput[:,i])**2)
                l2 = np.sqrt(up/down)
                self.assertLess(l2, 1e-12)
```

Reply:
Besides the style guide issue, I think you don't need to initialize up and down to zero.

Reply:
changed